src/main/resources/api/service_openapi.yaml

```diff
@@ -267,41 +234,49 @@ components:
       required: true
       schema:
         type: string
+        # format: uuid
```

Comment:
This is correct, but turning it on requires some actual changes to the java code, so I've left it commented out for now.